process/tls.cpp

```diff
@@ -176,14 +192,20 @@ bool TLSPlugin::parse_tls(const char *data, int payload_len, RecordExtTLS *rec)
    payload.data += tmp + 1; // Skip session id
 
    if (hs_type == TLS_HANDSHAKE_CLIENT_HELLO) {
-      // Process cipher suites
-      get_ja3_cipher_suites(ja3, payload);
-      if (!payload.valid) {
-         return false;
+      if (ja3) {
+         get_ja3_cipher_suites(*ja3, payload);
+         if (!payload.valid) {
+            return false;
+         }
+      } else {
+         if (payload.data + 2 > payload.end) {
+            return false;
+         }
+         payload.data += ntohs(*(uint16_t *) payload.data) + 2; // Skip cipher suites
       }
 
       tmp = *(uint8_t *) payload.data;
-      if (payload.data + tmp + 2 > payload.end) {
+      if (payload.data + tmp + 3 > payload.end) {
```

Comment:
This should definitely be explained in a comment! What does this sum means and why 3?